Readme.md

```diff
@@ -16,14 +16,15 @@ There are not a lot of frills yet, but I was able to swap out restify in a
 fairly complex app and have all its unit tests pass.  The app runs 40-50% more
 calls per second on restiq than before.
 
-        var Restiq = require('restiq');
-        var app = Restiq.createServer();
+    var restiq = require('restiq');
```

Comment:
Yeah, that was my thought. You expect people to not use `new Restiq`, so I made it lowercase.